.theme-check.yml

```
@@ -0,0 +1,3 @@
+# Explicitly enable all checks (by default, all are enabled)
+checks:
+  all: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)